docs/vault/guide.md

```diff
@@ -121,6 +121,22 @@ Now you must send the BTC to the Bitcoin address **generated by the parachain**.
 
 You Vault only starts earning rewards once BTC is locked - and rewards are determined by your share of the total BTC locked in the system. Hence, you can increase your rewards by bringing your own BTC into the system.
 
+### Atomic Collateral Deposit & Self-mint
+
+Sometimes you may prefer to _only_ self mint. However, it's possible that you receive issue requests shortly after registration or depositing additional collateral. To prevent this, you can use polkadot-js to do the two actions atomically.
+
+#### If vault is not yet registered
+
+If you have not registered the vault yet, you can atomically register with collateral and do the self-issue. To achieve this, first start your vault client without the `--auto-register` CLI argument. When the client starts up, it will register your public bitcoin key, which is a prerequisite to vault registration. Next, you use the polkadot.js app to do a batch of `registerVault` and `requestIssue`. The amount of collateral required to request a given number of IBTC/KBTC can be calculated as `BTC * secure_collateral_threshold * exchange_rate`. The secure collateral threshold for DOT/KSM at the time of writing is 2.6. The exchange rate can be found on the dashboard page on the Interlay/Kintsugi app. The collateral uses 10 digits for DOT (1 DOT  = 10000000000) and 12 digits for KSM (1 KSM = 1000000000000). If using other currencies, adapt accordingly. An example is shown below.
```

Comment:
```suggestion
If you have not registered the vault yet, you can atomically register with collateral and do the self-issue. 

1. First start your vault client without the `--auto-register` CLI argument. When the client starts up, it will register your public bitcoin key, which is a prerequisite to vault registration. 

2. Next, you use the polkadot.js app to do a batch of `registerVault` and `requestIssue`. Go to "Extrinsics", select `utility` > `batchAll` in the dropdowns. You can then add multiple calls that will be executed all at once (see screenshot below). 

?> The amount of collateral required to request a given number of IBTC/KBTC can be calculated as `BTC * secure_collateral_threshold * exchange_rate`. The secure collateral threshold for DOT/KSM at the time of writing is 2.6. The exchange rate can be found on the dashboard page of the Interlay/Kintsugi app. The collateral uses 10 digits for DOT (1 DOT  = 10000000000) and 12 digits for KSM (1 KSM = 1000000000000). If using other currencies, adapt accordingly. 
```